product_extended_segmentation/model/template.py

```diff
@@ -183,7 +183,11 @@ def _factor(factor, product_efficiency, product_rounding):
                     cr, uid, bom.product_uom.id, routing_price,
                     bom.product_id.uom_id.id)
                 price += routing_price
-                sgmnt_dict['production_cost'] += routing_price
+                try:
+                    fn = wc.segmentation_cost or 'production_cost'
+                except AttributeError:
+                    fn = 'production_cost'
+                sgmnt_dict[fn] += routing_price
```

Comment:
This module is not yet dependent on mrp_workcenter_segmentation
in the meantime while testing this peace of code avoid any depend on that module
when becomes dependent on that module this try-except will get rid.